pkg/service/roommanager.go

```diff
@@ -197,7 +197,10 @@ func (r *RoomManager) CloseIdleRooms() {
 	r.lock.RUnlock()
 
 	for _, room := range rooms {
-		room.CloseIfEmpty()
+		reason := room.CloseIfEmpty()
+		if reason != "" {
+			logger.Infow("closing idle room", "room", room.Name(), "reason", reason)
+		}
```

Comment:
if we logged this in `(*Room).CloseIfEmpty` we could use the room's logger - it already has the name/id